mars/services/scheduling/supervisor/globalslot.py

```diff
@@ -20,25 +20,24 @@
 
 from .... import oscar as mo
 from ....typing import BandType
+from ... import Resource, ZeroResource
 
 logger = logging.getLogger(__name__)
 
 
 class GlobalSlotManagerActor(mo.Actor):
     # {(address, resource_type): {(session_id, subtask_id): slot_id}}
```

Comment:
Comment `slot_id` need to be modified too.

Reply:
Good catch, I'll fix it